examples/uikits/src/pages/expo.js

```diff
@@ -1,39 +1,189 @@
+import { BlurView } from 'expo-blur'
+import { Camera } from 'expo-camera'
+import Constants from 'expo-constants'
+import * as Font from 'expo-font'
+import * as Location from 'expo-location'
+import * as Permissions from 'expo-permissions'
+import React, { useEffect, useState } from 'react'
+import { Button, ScrollView, Image, Text, View } from 'react-native'
 
-/*
-import React, { useState } from 'react'
-
-import Layout from '../components/layout'
 import Example from '../components/example'
-import { Text } from 'react-native'
-import { LinearGradient } from 'expo'
+import Layout from '../components/layout'
+
+function LocationExample() {
+  const [item, setItem] = useState(null)
 
-const LinearGradientExample = () => {
-  const [text, setText] = useState('')
   return (
-    <Example title="LinearGradient">
-      <LinearGradient
-        colors={['#4c669f', '#3b5998', '#192f6a']}
-        style={{ padding: 15, alignItems: 'center', borderRadius: 5 }}
+    <Example title="Location" style={{ justifyContent: 'space-around' }}>
+      <Button
+        title="Get Location"
+        onPress={async () => {
+          try {
+            setItem(await Location.getCurrentPositionAsync())
+          } catch ({ message }) {
+            alert('Something went wrong: ' + message)
+          }
+        }}
+      />
+      {item && <JSONView json={item} />}
+    </Example>
+  )
+}
+
+function JSONView({ json }) {
+  return (
+    <ScrollView style={{ flex: 1, overflow: 'scroll' }}>
+      <Text
+        style={{
+          backgroundColor: 'transparent',
+          fontSize: 15,
+          color: '#000',
+        }}
       >
+        {JSON.stringify(json, null, 2)}
+      </Text>
+    </ScrollView>
+  )
+}
+
+function FontExample() {
+  const [loaded, setLoaded] = useState(false)
+
+  useEffect(() => {
+    ;(async () => {
```

Comment:
; is a typo? or is this useful for some reason?

Reply:
I think it's a new ESLint bug going around, will remove.